plugins/shared-react/src/types/pipeline/pipelineRun.ts

```diff
@@ -87,7 +87,8 @@ export type PipelineRunStatus = {
   skippedTasks?: {
     name: string;
   }[];
-  pipelineResults?: TektonResultsRun[];
+  pipelineResults?: TektonResultsRun[]; // in tekton v1 pipelineResults is renamed to results
```

Comment:
Based on this comment, is this a good time to rename the type so it's aligned with v1?

Reply:
We are trying to support both so I've left both here and check for either when looking for results.